delayed-state-rendering.md

```diff
@@ -0,0 +1,273 @@
+- Feature Name: Delayed State Rendering
+- Start Date: 2019-06-21
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+- Author: Joseph Nix, nixjdm at terminallabs dot com, [nixjdm@GitHub](https://github.com/nixjdm)
+
+# Summary
+
+This SEP proposes a way of delaying rendering of portions of states until after other states have executed.
```

Comment:
The `of delaying rendering of portions of states` is a bit of a tongue twister. Its written correctly but every time I read it I mess it up and have to slowly re-read it to correct myself.

I would replace with `to delay rendering of selected states` or `to delay rendering`.